### ISSUE TITLE
Train loss become nan at the begining with CULane data

### ISSUE THREAD

**Issue by FCInter** [harryhan618/SCNN_Pytorch#35]
I download the code on April 18, 2020. I configured the data path as is required. Then I launch the traning process by:

`python3 train.py`

After like 2 minutes, just 100 iterations, the batch loss became `nan`.

My only modification to the code was in the `model.py`, `class SCNN`, I replace `self.bce_loss = nn.BCELoss()` with `self.bce_loss = nn.BCEWithLogitsLoss()`, because the former will cause errors and I found a solution doing this.

Except this, I did not change any single word of the code.

What's wrong with my reimplementation?

I'm looking forward to your kind reply.

Thank you very much!!!

**Comment by harryhan618** [comment]
Well, since you replace `self.bce_loss = nn.BCELoss()` with `self.bce_loss = nn.BCEWithLogitsLoss()`, there must be somewhere in the code that does `sigmoid` operation that you should modify.
